jupyterlab-ihaskell/src/codemirror-ihaskell.ts

```diff
@@ -1,31 +1,35 @@
-import * as CodeMirror from 'codemirror';
+import { ICodeMirror } from '@jupyterlab/codemirror';
 
 import 'codemirror/lib/codemirror';
```

Comment:
you probably want to clobber _all_ these `lib/codemirror` imports...

Reply:
Updated!